docs/GetStarted/getting_started_android_cmake.md

```diff
@@ -218,3 +218,50 @@ problems for Vulkan device enumeration under `/data/local/tmp/`. A known
 workaround is to copy the `libGLES_mali.so` library under `/data/local/tmp/` and
 rename it as `libvulkan.so` and then prefix `LD_LIBRARY_PATH=/data/local/tmp`
 when invoking IREE executables.
+
+### Dylib LLVM AOT backend
+
+To compile iree module for the target android device (assume aarch64,android 29)
+we need install standalone toolchain and setting AOT linker path environment
+variable:
+```shell
+$ export ANDROID_ARM64_TOOLCHAIN=<path to put new arm64 toolchain>
+$ $ANDROID_NDK/build/tools/make-standalone-toolchain.sh --arch=arm64 --platform=android-29 \
+    --install-dir=$ANDROID_ARM64_TOOLCHAIN
+$ export IREE_LLVMAOT_LINKER_PATH=$ANDROID_ARM64_TOOLCHAIN/aarch64-linux-android/bin/ld
```

Comment:
Is this `IREE_LLVMAOT_LINKER_PATH` path to be read by `iree-translate` to invoke the linker under the hood? Is it possible to provide a "default" value for it when compiling `iree-translate`? When cross compiling towards Android, we already specified the `ANDROID_ABI` and `ANDROID_PLATFORM`. Together with `android.toolchain.cmake`, we should be able to figure out which `ld` to use by using the [`ANDROID_TOOLCHAIN_PREFIX`](https://android.googlesource.com/platform/ndk/+/master/build/cmake/android.toolchain.cmake#399) cmake variable. so here we are requiring to provide the information again and making a standalone toolchain, which is a bit annoying. Particularly because [standalone toolchain](https://developer.android.com/ndk/guides/standalone_toolchain
) has a big "obsolete" in its title..

Not really blocking, but want to see what you think here. :)

Reply:
Chatted with @antiagainst offline.  Very good idea I will file an issue to forward android.toolchain.cmake linker path when building host tools.